spec_1/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "the_guessing_game",
```

Comment:
The idea is to have everything be a standalone module so that if you're reviewing you could just npm i within the particular spec and not have to install every dependency if you're just working with one spec.